collections/Gr_class/MS_Gr_class_c_223_P_a.xml

```diff
@@ -81,6 +83,7 @@
       <revisionDesc>
          <change when="2017-07-01">First online publication.</change>
          <change when="2017-05-25"><persName>James Cummings</persName> Up-converted the markup using <ref target="https://github.com/jamescummings/Bodleian-msDesc-ODD/blob/master/convertTolkien2Bodley.xsl">https://github.com/jamescummings/Bodleian-msDesc-ODD/blob/master/convertTolkien2Bodley.xsl</ref></change>
+         <change when="2025-10-21"><persName>Peter Toth</persName>Updated after digitisation.</change>
```

Comment:
I'm not sure but this might need to be at the top, before the other `<change>` elements. Can you add your name in the `respStmt` at the beginning of the file as well if you haven't already?  This applies to all the files...